README.md

```diff
@@ -243,7 +244,15 @@ func main() {
 		c.FullPath() == "/user/:name/*action" // true
 	})
 
-	router.Run(":8080")
+  // This handler will add a new router for /user/groups.
+  // Exact routes are resolved before param routes, regardless of the order they were defined.
+  // Routes starting with /user/groups are never interpreted as /user/:name/... routes
+  router.GET("/user/groups", func(c *gin.Context) {
+    c.String(http.StatusOK, "The available groups are [...]", name)
+  })
+
+
+router.Run(":8080")
```

Comment:
please check the code format.

Reply:
thanks, i didn't think about `gofmt` in the readme.
fixed in 89f4ae2dd3369a6ee503c593d6cb287d8ec36b85 (which was actually a squashed commit per the PR guidelines)